src/main/java/net/sf/jabref/gui/journals/ManageJournalAbbreviationsView.java

```diff
@@ -0,0 +1,315 @@
+
+import net.sf.jabref.JabRefException;
+import net.sf.jabref.gui.FXAlert;
+import net.sf.jabref.gui.FXDialogs;
+import net.sf.jabref.gui.IconTheme;
+import net.sf.jabref.logic.journals.Abbreviation;
+import net.sf.jabref.logic.l10n.Localization;
+
+import com.airhacks.afterburner.views.FXMLView;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ButtonType;
+import javafx.scene.control.ComboBox;
+import javafx.scene.control.DialogPane;
+import javafx.scene.control.TableCell;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.GridPane;
+import javafx.scene.layout.Priority;
+import javafx.scene.text.Text;
+import javafx.stage.FileChooser;
+import javafx.stage.Stage;
+import javafx.util.Callback;
+import javafx.scene.control.Alert.AlertType;
+
+public class ManageJournalAbbreviationsView extends FXMLView {
+
+    private final ManageJournalAbbreviationsViewModel viewModel = new ManageJournalAbbreviationsViewModel();
+
+    @FXML
+    private TableView<Abbreviation> journalAbbreviationsTable;
+    @FXML
+    private TableColumn<Abbreviation, String> journalTableNameColumn;
+    @FXML
+    private TableColumn<Abbreviation, String> journalTableAbbreviationColumn;
+    @FXML
+    private TableColumn<Abbreviation, String> journalTableEditColumn;
+    @FXML
+    private TableColumn<Abbreviation, String> journalTableDeleteColumn;
+    @FXML
+    private Button cancelButton;
+    @FXML
+    private Button saveButton;
+    @FXML
+    private Button addAbbreviationButton;
+    @FXML
+    private ComboBox<AbbreviationsFile> journalFilesBox;
+    @FXML
+    private Button addJournalFileButton;
+    @FXML
+    private Button addNewJournalFileButton;
+    @FXML
+    private Button removeJournalAbbreviationsButton;
+
+
+    public ManageJournalAbbreviationsView() {
+        super();
+        bundle = Localization.getMessages();
+    }
+
+    @FXML
+    private void initialize() {
+        setUpTable();
+        setBindings();
+        viewModel.createFileObjects();
+        journalFilesBox.getSelectionModel().selectLast();
+    }
+
+    private void setUpTable() {
+        journalTableNameColumn.setCellValueFactory(cellData -> cellData.getValue().nameProperty());
+        journalTableAbbreviationColumn.setCellValueFactory(cellData -> cellData.getValue().abbreviationProperty());
+        journalTableEditColumn
+                .setCellFactory(new Callback<TableColumn<Abbreviation, String>, TableCell<Abbreviation, String>>() {
+
+                    @Override
+                    public TableCell<Abbreviation, String> call(TableColumn<Abbreviation, String> stringListView) {
+                        return new TableCell<Abbreviation, String>() {
+
+                            @Override
+                            protected void updateItem(String s, boolean b) {
+                                super.updateItem(s, b);
```

Comment:
Try using  a lambda for the callback, makes the code more readable:
http://stackoverflow.com/questions/23978676/convert-callback-into-java-8-lambda-expression

Please use more speaking  variable names for s and b, for example like here:
http://code.makery.ch/blog/javafx-8-tableview-cell-renderer/